frontend/src/app/corpus-info/corpus-info.component.ts

```diff
@@ -32,7 +32,7 @@ export class CorpusInfoComponent implements OnInit {
 
     setCorpus(corpus: Corpus) {
         this.corpus = corpus;
-        this.documentation$ = this.apiService.corpusDocumentationPages(corpus).pipe(
+        this.documentation$ = this.apiService.corpusDocumentationPages(corpus.name).pipe(
```

Comment:
I thought we moved away (as much as possible) from using corpus names? Can we use ID instead? With user uploaded corpora I foresee the unique corpus name restraint to be dropped in the future.